IceNet/polar-modelling-icenet.ipynb

```diff
@@ -0,0 +1,1411 @@
+{
```

Comment:
<p>Very nice!! This interactive plot is great!!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
<p>Thanks, Tom and I tried one with <code>hvplot</code> but it wasn't giving the most aesthetic results. We ended using <code>panels</code> which is a great python library for generating dashboard and support <code>Matplotlib</code> plots.  </p>

Reply:
<p>Thanks Nick!</p>